WordPress/Helpers/ContextHelper.php

```diff
@@ -129,6 +129,10 @@ final class ContextHelper {
 	 */
 	public static function has_object_operator_before( File $phpcsFile, $stackPtr ) {
 		$tokens = $phpcsFile->getTokens();
+		if ( isset( $tokens[ $stackPtr ] ) === false ) {
```

Comment:
This was written as `isset === false` on purpose? Because 

```suggestion
		if ( !isset( $tokens[ $stackPtr ] ) ) {
```

would yield the same result, no?

Reply:
Yes, would yield the same result, I'm just used to PHPCS, where comparisons must be explicit and strict, which `!` is not.
I've gotten to like these explicit comparisons as it make the code much easier to scan as the `!` is easily overlooked.